### ISSUE TITLE
`divmod` no longer type checks in 1.1.213

### ISSUE THREAD

**Issue by GBeauregard** [microsoft/pyright#2917]
**Describe the bug**
Since 1.1.213 some uses of the `divmod` function no longer type check.

**To Reproduce**
```python
# pyright: strict
def sec_to_str(seconds: float) -> str:
    a = divmod(abs(seconds), 60)
    return str(a)
```

**Expected behavior**
The code should type check

**VS Code extension or command-line**
commandline 1.1.213

**Additional context**
If you change it to `60.` it type checks okay, but the typeshed for `divmod` looks all right at a glance. It also type checks if you remove the `abs()`

Original function:
```python
def sec_to_str(seconds: float) -> str:
    m, s = divmod(abs(seconds), 60)
    h, m = divmod(m, 60)
    return f"{'-' if seconds<0 else ''}{int(h):02d}:{int(m):02d}:{int(s):02d}"
```

**Comment by erictraut** [comment]
Thanks for reporting. This is a bad regression, so I'm going to work on a fix and do another release before it impacts other pyright users.

**Comment by erictraut** [comment]
This is fixed in 1.1.214, which I just published. Apologies for any inconvenience.
